appveyor.yml

```diff
@@ -0,0 +1,29 @@
+version: 0.1.{build}
+pull_requests:
+  do_not_increment_build_number: true
+branches:
+  only:
+  - dev
+  - master
+
+image:
+- Ubuntu
+- Visual Studio 2017
+
+max_jobs: 1
+
+configuration: Debug
+
+install:
+  - pwsh: Install-Module PSDepend -Force -Scope CurrentUser
```

Comment:
I don't see `dotnet SDK 2.1` installed. Is it pre-installed in the appveyor images?

Reply:
Yeah it comes by default.